tests/test-sessions/test-mongodb-sessions/src/test/java/org/eclipse/jetty/nosql/mongodb/PurgeValidSessionTest.java

```diff
@@ -123,6 +123,75 @@ public void testPurgeValidSession() throws Exception
     }
 
 
+    @Test
+    public void testPurgeValidSessionWithPurgeLimitSet() throws Exception
```

Comment:
Worth noting that I ran that test as part of the whole suite of tests, not by itself. Maybe some of the other tests add sessions that don't expire.